api/v1alpha1/pattern_types.go

```diff
@@ -163,6 +164,9 @@ type PatternStatus struct {
 	ClusterVersion string `json:"clusterVersion,omitempty"`
 	//+operator-sdk:csv:customerresourcedefinitions:type=conditions
 	Conditions []PatternCondition `json:"conditions,omitempty"`
+	// Health status of the generated argoCD application
+	//+operator-sdk:csv:customresourcedefinitions:type=status
+	AppHealthStatus health.HealthStatusCode `json:"appHealthStatus,omitempty"`
```

Comment:
There are at least two applications per pattern, usually more. So an array would be needed here